src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="container">
+      <div class="search-bar search-bar__big"
+        data-qa="big"
+        >
+        <input
+         type="text"
+         data-qa="keypress"
+         value="Try “Los Angeles”"
+        >
+      </div>
+      <div class="search-bar search-bar__small"
```

Comment:
The same as about the previous my comment

Reply:
your colleagues recomended breaking it down line by line. It is the screenshots above.

Reply:
and exactly what approach are you using now?